Automattic/newspack-theme
# Linting and formatting - JS code

### All Submissions:






### Changes proposed in this Pull Request:



This sets up Eslint for linting JS files and Prettier for formatting code.
All linting errors have been fixed and code has been formatted. 
Mostly these were autofixes and obvious cases like unused variables. 

In one file I've removed a code block, as I did not see it used - but maybe I did not test it right:
https://github.com/Automattic/newspack-theme/compare/fix/linting-and-formatting-js?expand=1#diff-6b0bd84550cf2a44d5103a8fd38adcadL25-R25
So I'd appreciate a closer look there @laurelfulford ;) 

### How to test the changes in this Pull Request:

1. run `npm run lint:js` - it should not throw any errors
2. run `npm run format:js` - it should not change any files

### Other information:
